src/Components/MiniPaletteArea/MiniPaletteArea.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
Nice work! I think this makes more sense now having the MiniPaletteArea as a component and the MiniPalette as a container